### PR TITLE
Add Container Operations Benchmark

### PR DESCRIPTION
**Pull request by Helen-Xie**
part of #25  Add a few benchmark test cases about Container

Signed-off-by: Helen-Xie <yulinxie@zju.edu.cn>

### CODE REVIEW

**Comment by Helen-Xie** [comment]
ping @feiskyer

File: pkg/benchmark/container.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		})
+
+		Measure("benchmark about creating Container", func(b Benchmarker) {
+			var containerID string
+
+			operation := b.Time("create Container", func() {
+				containerID = framework.CreateDefaultContainer(rc, ic, podID, podConfig, "Container-for-creating-benchmark-")
+			})
+
+			Expect(operation.Seconds()).Should(BeNumerically("<", 2), "create Container shouldn't take too long.")
```
Comment (feiskyer) [inline pkg/benchmark/container.go]: Should we measure those by ms instead of second?

cc/ @xlgao-zju

File: pkg/benchmark/container.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+			rc.StopContainer(containerID, framework.DefaultStopContainerTimeout)
+			rc.RemoveContainer(containerID)
```
Comment (feiskyer) [inline pkg/benchmark/container.go]: create/start/stop/remove has already occurred in every measure cases. Could we merge all those into same Measure block (but different operation)?

Comment (Helen-Xie) [inline pkg/benchmark/container.go]: update

File: pkg/benchmark/container.go (hunk 1 above)
Comment (Helen-Xie) [inline pkg/benchmark/container.go]: there is no func `Millisecond()` in operation, so I still use the `Seconds()`.

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**